### PR TITLE
Enhancement for update restock product

### PR DESCRIPTION
**Pull request by MeghaY**
when restock-level is 0 don't run the script to update product, when restock-level is 1, set reorder-point to 1 and when restock-level is 2 or more, set reorder-point to one less than restock-level

### CODE REVIEW

File: commands/generate-stock-order.js [hunk 1]
```diff
@@ -628,23 +628,40 @@ var runMe = function(connectionInfo, orderName, outletId, supplierId, since, gen
...
+                      "reorder_point": reorderPoint,
+                      "restock_level": product.orderMore
+                    }
+                  ]
+                };
+
+                //update call to Vend API product endpoint along with new values for reorder point and restock level
+                return vendSdk.products.update({apiId:{value: product.id},body:{value: updateData}},connectionInfo)
+                  .then(function(response){
+                    console.log('Response from update product: '+ JSON.stringify(response,null,2));
```
Comment (pulkitsinghal) [inline commands/generate-stock-order.js:661]: will you comment this out at some point?

File: commands/generate-stock-order.js [hunk 2]
```diff
@@ -628,23 +628,40 @@ var runMe = function(connectionInfo, orderName, outletId, supplierId, since, gen
               console.log('product: '+ JSON.stringify(product,null,2));
```
Comment (pulkitsinghal) [inline commands/generate-stock-order.js:628]: will you comment this out at some point?

Comment (MeghaY) [inline commands/generate-stock-order.js:628]: yes to both.. That was just for testing purpose.

File: commands/generate-stock-order.js [hunk 3]
```diff
@@ -628,23 +628,40 @@ var runMe = function(connectionInfo, orderName, outletId, supplierId, since, gen
...
-              //update call to Vend API product endpoint along with new values for reorder point and restock level
-              return vendSdk.products.update({apiId:{value: product.id},body:{value: updateData}},connectionInfo)
-                .then(function(response){
-                  //console.log('Response from update product: '+ JSON.stringify(response,null,2));
-                  return Promise.resolve();
-                })
+              //when restock-level(product.orderMore) is 0 don't run that script
+              // when restock-level is 1, set reorder-point to 1
+              // when restock-level is 2 or more,set reorder-point to one less than restock-level
+              if(product.orderMore === 0){
```
Comment (pulkitsinghal) [inline commands/generate-stock-order.js:634]: I don't think `orderMore` can be negative but just for safety should we also have a check to enter this block if its negative?

**Comment by pulkitsinghal** [comment]
@MeghaY - Overall i think it is fine and nothing really needs to be changed, very minor comments inline.

**Pull request merged**